qtfred/src/ui/dialogs/ReinforcementsEditorDialog.cpp

```diff
@@ -0,0 +1,236 @@
+#include "ReinforcementsEditorDialog.h"
+#include "ui_ReinforcementsDialog.h"
+
+#include <globalincs/linklist.h>
+#include <ui/util/SignalBlockers.h>
+#include <QCloseEvent>
+#include <QListWidget>
+#include <qlineedit.h>
+
+constexpr int INTMAX = 2147483647;
```

Comment:
Isn't this constant available from somewhere?

Reply:
I couldn't find one that wasn't part of a Microsoft header, and I didn't want to add a header just for that.

Reply:
There are many places in the codebase that refer to `INT_MAX`, so it's defined somewhere in a cross-platform way.  Try `#include "globalincs/pstypes.h"`.